.travis.yml

```diff
@@ -1,40 +1,39 @@
 services:
 - docker
+- mongodb
 
 branches:
   only:
     - master
-    - module_10
+    - module_11
```

Comment:
We would like to run tests on all branches, but only **deploy** (i.e. pushing `todo-app:latest`  and hitting the webhook) on master.

You can either use Travis's `deploy` section instead of putting it inside `script`, or you need to check the current branch.

Note that if you are using "deploy", then you will probably want to put your commands in an .sh file (because Travis double processes the deploy and the dollar sign in the value will cause problems)

Reply:
Will look into this as part of module 12